akka-distributed-data/src/main/scala/akka/cluster/ddata/Replicator.scala

```diff
@@ -1111,6 +1115,10 @@ final class Replicator(settings: ReplicatorSettings) extends Actor with ActorLog
     }
   }
 
+  def receiveWeaklyUpMemberUp(m: Member): Unit =
+    if (matchingRole(m) && m.address != selfAddress)
+      weaklyUpNodes += m.address
+
   def receiveMemberUp(m: Member): Unit =
     if (matchingRole(m) && m.address != selfAddress)
       nodes += m.address
```

Comment:
i miss the path. it is right, when the weakly node become normal Up node, it remove from the weaklyUpNodes. i'll fix it.